### ISSUE TITLE
Preview fails when python.pythonPath contains ${workspaceFolder}

### ISSUE THREAD

**Issue by segevfiner** [vscode-restructuredtext/vscode-restructuredtext#99]
**Environment**
- Extension version: 70.0.0 
- Visual Studio Code version: 1.26.1
- Operating System version: Windows 10.0.17134.228 x64
- Python version: 3.7.0 x64
- sphinx version: 1.7.8
- A sample of conf.py or its important fragments: A default one from `sphinx-quickstart`

When `python.pythonPath` contains `${workspaceFolder}`, the auto detected `restructuredtext.sphinxBuildPath` will not expand that variable, leading to a build failure. Such a configuration is automatically created by the VS Code Python extension when selecting a Python interpreter that is under the workspace folder, such as a virtualenv.

See https://github.com/Microsoft/vscode/issues/57551

**Expected behavior**
The build should work in that case without having to edit the configuration.

**Actual behavior**
It fails:
```
Error
Command failed: ${workspaceFolder}\venv\Scripts\python.exe -m sphinx -b html . "...\docs\_build\html"
The system cannot find the path specified.

Error: Command failed: ${workspaceFolder}\venv\Scripts\python.exe -m sphinx -b html . "...\docs\_build\html"
The system cannot find the path specified.

	at ChildProcess.exithandler (child_process.js:287:12)
	at emitTwo (events.js:126:13)
	at ChildProcess.emit (events.js:214:7)
	at maybeClose (internal/child_process.js:925:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)

The system cannot find the path specified.
```

**Detailed Steps to reproduce the problem**
1. Create a Sphinx project and a virtualenv in the same VS Code workspace folder.
2. Set the VS Code Python extension interpreter to the virtualenv interpreter using the `Python: Select Interpreter` command so that it contains `${workspaceFolder}`.
3. Try to preview.

**Comment by lextm** [comment]
Thanks for the detailed report. I will see into it.

**Comment by lextm** [comment]
I just shipped 71.0.0 release. Please test the fix. Thanks.

**Comment by segevfiner** [comment]
It's working for me. Thanks!
